### ISSUE TITLE
Extend personal-best dive try for magnification 10^500

### ISSUE THREAD

**Issue by ckormanyos** [ckormanyos/mandelbrot#104]

**Comment by ckormanyos** [comment]
See also #19 and the methods of #100

**Comment by S-Streulicht** [comment]
I think you could go directly to values of e-1000 for deep zoom, since the calculation time does only scale (significantly) with the number of iterations but not with the zoomlevel any more. I suggest to perform small images like 64 by 64 to get a feeling for the iterations. And the precision of the central point.
at zomlevels higher then e-1000 the central point of the image (20_zoom_very_deep) needs to be refined, I guess

**Comment by ckormanyos** [comment]
> go directly to values of e-1000 for deep zoom,

Indeed, but I want two intermediate pics for the gallery, one around $10^{300}$ and a secont around $10^{500}$. Then I'll go for a cool pic around $10^{1000}$.

**Comment by ckormanyos** [comment]
@S-Streulicht then we need to find new iteration points... since I've got a couple around $10^{100}$, but I'm experiencing difficulty getting thousand-order points of interest.

**Comment by S-Streulicht** [comment]
> @S-Streulicht then we need to find new iteration points... since I've got a couple around $10^{100}$, but I'm experiencing difficulty getting thousand-order points of interest.

Hm, Finding points there could be 3 trivial ones on the real axis:
The very left one atthe tip of the mandelbrot and then veryright one at the butt I dident test it but the exact point should be easyto find in an iterative way pic twp oint on the real axis onel outode mandelbrot one inside than use a binary search to narrow the point.

The third one 9s abit more tricky but I expect there is tonth of stuf to be found in the net: at some point the mandelbrot isself similar to itself no mater the deep you zoom it is always lpoking the same If i remember it is not realy one point but it changes with the zoomlevel

**Comment by ckormanyos** [comment]
> Finding points

Hi @S-Streulicht I'm working on a visual semi-automated _discovery_ program. When the GPU accelration comes in, this should allow for point-and-calculate that should work relatively quickly.

**Comment by ckormanyos** [comment]
Hi @S-Streulicht here is a very, very, very preliminary pic of MandelbrotDiscovery in action, as I zoom into a point having 60 digits of magnification.

![grafik](https://github.com/ckormanyos/mandelbrot/assets/2404721/da9f975b-8849-426d-8c7e-85852e353b44)

**Comment by ckormanyos** [comment]
Hi @S-Streulicht I ended up going outsinde for a bike ride and hit the tavern then this is the point I ended up with on that particular run. Cool find. Although I was hoping for a hundred orders.

![mandelbrot_discovery_in_action2](https://github.com/ckormanyos/mandelbrot/assets/2404721/d1a35660-02b0-43e1-830c-6093fb3ccf77)

**Comment by S-Streulicht** [comment]
> > Finding points
> 
> Hi @S-Streulicht I'm working on a visual semi-automated _discovery_ program. When the GPU accelration comes in, this should allow for point-and-calculate that should work relatively quickly.

@ckormanyos can you define "interesting point"?
yust a point which can be zoomed in very deep, or does it need to have some picture feature?
If it can be a random point yust next to the mandelbrotset, like the üictures of your galerie we have some interesting feature set:
1. The center should have the higest iteration
2. zooming in deper and deeper you will find a mini mandelbrot set.

Finding the point of interest:
I don't think you need the full picture to detern the center for the next zoom level, I guees it is just fine to calculate the the pixel on axist (vert and hor) in near the center you will find a platoo (maximum iterations alowed) For two 1D curves the start and the end of the platoo is easy to find. -> center of platoo is the next center of the investigation. Next step increase the iteration and calculate the pixel on axis again for just the region of the platoo. Repeed until the platoo dosen't shrink any more. At that point we have two options: First the number of iterations is just not high enough or we have a mandelbrot

**Comment by ckormanyos** [comment]
> For two 1D curves the start and the end of the platoo is easy to find. -> center of platoo is the next center of the investigation.

Hi @S-Streulicht so far I've only been able to find points randomly. I would like to calculate or predict them. But I've not figured out how to do that. I've heard about some orbit calculations but never pin-pointed them.

**Comment by ckormanyos** [comment]
@S-Streulicht yes, indeed, we have collected a few. But I'd like a just several more, let's say, pics exhibiting even greater amounts of _fluffiness_. I can find fluffy around order $10^30$ with any given random search. Also, at times I find some stark harsh B/W pics with magnifications as high as I want to go in order.

But I'm experiencing difficultiies finding interesting pics with hundreds of orders that offer _fluffy_ details. I do not know what actual equations govern such orbits.

I don't know if this makes any sense?

**Comment by ckormanyos** [comment]
> can you define "interesting point"?

Hi @S-Streulicht that is very subjective and I've changed my views as I dive. I seek points having many details, not just stark contr4asts, yet require a bit-float to obtain.

This recent pic I got was great. See pic below. I would have liked a hundred orders, but that does not really matter. Then I'd like some pics that are not exactly symmetrical, but have some kind of curl too.

![Uploading mandelbrot_MANDELBROT_80_LOW_DOWN_HALO_01.jpg…]()
